charts/tempo-distributed/templates/compactor/deployment-compactor.yaml

```
@@ -0,0 +1,61 @@
+{{- if .Values.compactor.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "tempo.compactorFullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "tempo.compactorLabels" . | nindent 4 }}
+  {{- with .Values.compactor.annotations }}
+  annotations:
+    {{ toYaml . | indent 4 }}
+  {{- end }}
+spec:
+  minReadySeconds: 10
+  replicas: {{ .Values.compactor.replicas }}
+  revisionHistoryLimit: 10
```

Should not be hard-coded.
```suggestion
  revisionHistoryLimit: 10
```